packages/modules/src/modules/banners/momentTemplate/stories/WithChoiceCardsHeaderImage.tsx

```diff
@@ -12,7 +12,6 @@ export const BannerWithChoiceCardsHeaderImage = bannerWrapper(
             backgroundColour: '#F1F8FC',
         },
         headerSettings: {
-            textColour: '#0077B6',
```

Comment:
I believe the `textColour` prop had been removed from these stories in respect to the rendering of the banner header. Though depending on implementation this prop could still be optional and therefore removed

Reply:
Have left out, because matches the `showHeader: { text: false, image: true }` on line above now.